docs/concepts.rst

```diff
@@ -0,0 +1,128 @@
+Concepts
+########
+
+
+Basically, *Kinto* is a service where client applications can store and retrieve JSON data.
+
+In order to provide synchronization and sharing features for these data, *Kinto*
+introduces some basic concepts.
+
+*Kinto* objects in brief :
+
++-----------------+---------------------------------------------------------+
+| Object          | Description                                             |
++=================+=========================================================+
+| **bucket**      | Buckets can be seen as namespaces:                      |
+|                 | collections names won't collide if stored in different  |
+|                 | buckets.                                                |
++-----------------+---------------------------------------------------------+
+| **collection**  | A collection of records                                 |
++-----------------+---------------------------------------------------------+
+| **record**      | The actual stored data                                  |
++-----------------+---------------------------------------------------------+
+| **group**       | A named list of :term:`principals <principal>`.         |
++-----------------+---------------------------------------------------------+
+
+
+.. _concepts-buckets-collections-records:
+
+Buckets, Collections and Records
+================================
+
+A **record** is the smallest unit of data. By default, there is no schema,
+and the JSON can contain anything.
+
+A **collection** is a group of records. Records are manipulated as a list
+and can be filtered or sorted. Clients can obtain the list of changes that
+occured on the collection records since a certain revision (e.g. *last synchronization*).
+
+A **bucket** is an abstract notion used to organize collections and their
+permissions. A bucket named ``default`` is provided, whose collections and records
+are accessible to the current user only.
+
+.. image:: images/concepts-general.png
+
+Every kind of object manipulated by *Kinto* shares some common properties:
+
+* a unique identifier;
```

Comment:
While semi-colons can be used in lists, this is generally only the case in "complex" lists (i.e. lists where each item is a list in itself). There is another case where lists and semi-colons are used, but this is neither the time nor the place for an excursion deep into the jungle of English grammar. :wink: 

tl;dr These list items should be terminated with periods (and start with capital letters).